app/src/main/java/com/example/notes/model/NotesRemoteDBHelper.kt

```diff
@@ -6,13 +6,13 @@ import retrofit2.Response
 
 class NotesRemoteDBHelper {
```

Comment:
is NotesRemoteDBHelper calls the retrofit service ?
if that so, it shouldn't called `NotesRemoteDBHelper`